skops/io/_general.py

```diff
@@ -528,6 +529,37 @@ def _construct(self):
         return content_bytearray
 
 
+def operator_func_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
+    _, attrs = obj.__reduce__()
```

Comment:
instead of this, did you test what happens if we use the reduce loader we already have?